src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    int DEFAULT_LENGTH = 10;
```

Comment:
This constant is used as a value of created instances. So it's just a constant of FigureSupplier